README.md

```diff
@@ -200,6 +205,8 @@ You can install the `cega` chart via Helm CLI:
 helm install --version <chart-version> --name cega --namespace localega -f values.yml ega-charts/cega
 ```
 
+The fake cega uses `fake_cega` as password both for the RabbitMQ and the CEGA-Users services.
```

Comment:
Maybe we could reinforce that this is for testing purposes only and we recommend that people use `LocalEGA-deploy-init` for generating them, it is already mentioned at the beginning of readme how to do it.

Reply:
Well fake-cega is for testing, if some one will run it as a production service they need to use the stuff from the init script. This is mainly for simplifying CI stuff.